apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -577,6 +577,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
       },
       renderer: opt_options.renderer || Renderers.DEFAULT,
       comments: false,
+      media: '/blockly/media/google_blockly',
```

Comment:
For my understanding - where are the files actually stored? I thought `blockly/media` was an alias for `apps/static`?

Reply:
Yeah the top-level dir named blockly is definitely confusing, I think probably relates to the fact that the original (and current) name for the apps project is is "blockly-mooc". Given that naming, it would probably make the most sense for these assets to go in the top level `blockly/media/` directory (no subdirectory), but we already have assets for our CDO Blockly fork in there with the same names (`click.mp3`, etc) as files I want to copy here. Anyway, yes feels messy, but I couldn't think of a better approach without a larger refactor.